### PR TITLE
SupportSnapshots: Rename to "Get help"

### PR DESCRIPTION
**Pull request by ryantxu**
In trying to explain this to various folks, the feature name seems the most confusing.  Initially I avoided using something as general as "Get help" because it feels like it should then have a much more clear way to ask for help.

But maybe it is better to put something small and improve rather than introduce a new complicated word "Support snapshot"

This moves things under "More..." where UX feedback originally suggested :)

Wording still needs some ❤️ but I think it is generally better.  See also https://github.com/grafana/grafana/pull/55170

### CODE REVIEW

File: public/app/features/dashboard/components/SupportSnapshot/SupportSnapshot.tsx [hunk 1]
```diff
@@ -80,10 +80,14 @@ export function SupportSnapshot({ panel, plugin, onClose }: Props) {
...
             <FeatureBadge featureState={FeatureState.beta} />
+            &nbsp;
+            <a href="https://grafana.com/docs/grafana/latest/troubleshooting/" target="blank">
+              https://grafana.com/docs/grafana/latest/troubleshooting/
+            </a>
           </span>
           <span className="muted">
-            A support snapshot contains the query response data and raw panel settings. Include this snapshot in support
-            requests to help identify issues faster
+            When requesting help, including this snapshot a support snapshot contains the query response data and raw
```
Comment (chri2547) [inline public/app/features/dashboard/components/SupportSnapshot/SupportSnapshot.tsx:89]: Consider this wording: "To request troubleshooting help, send a snapshot of this panel to Grafana Labs Technical Support. The snapshot contains query response data and panel settings."

**Comment by torkelo** [comment]
Small tweak to header
Screenshot from 2022-09-15 19-36-34

File: pkg/setting/setting_plugins.go [hunk 2]
```diff
@@ -27,21 +27,27 @@ func extractPluginSettings(sections []*ini.Section) PluginSettings {
 func (cfg *Cfg) readPluginSettings(iniFile *ini.File) error {
 	pluginsSection := iniFile.Section("plugins")
 	cfg.PluginsEnableAlpha = pluginsSection.Key("enable_alpha").MustBool(false)
+	cfg.PluginNavLocation = pluginsSection.Key("nav_location").MustString("")
```
Comment (ryantxu) [inline pkg/setting/setting_plugins.go:30]: @torkelo is this an accidental commit to this PR?  seems unrelated

Comment (torkelo) [reply pkg/setting/setting_plugins.go:30]: Oops forgot I had some changes going when switching to this PR! Sorry!

**Review by torkelo — APPROVED** [review]
